docs/table_design/Data_distribution.md

```diff
@@ -1,107 +1,209 @@
 # Data distribution
 
-When you create a table, you can specify the data distribution method by configuring the partitioning and bucketing strategy in the table. An appropriate data distribution method helps evenly distribute data across the nodes of your StarRocks cluster. It can also reduce the data scanned and make full use of the concurrency of the cluster during queries, thereby improving query performance.
+Configuring appropriate partitioning and bucketing at table creation data can help to achieve even data distribution and improve query performance. Even data distribution means dividing the data into subsets according to certain rules and distributing them evenly across different nodes. It can also reduce the data scanned and make full use of the concurrency of the cluster during queries, thereby improving query performance.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/1102a91f-36a5-4166-9d5c-389a2d30db8a)
Within this short paragraph, "improve query performance" is mentioned twice. And based on the logic of this paragraph, "improve query performance" is actually the benefit brought by even data distribution. So, putting "even data distribution" and "improve query performance" in parallel in the first sentence is not a good choice, I think. I suggest removing "improve query performance" in the first paragraph, and retain "thereby improving query performance" in the end of this paragraph, as this can lead the user into a more natual logic that even data distribution brings query performance improvement.